src/Card/Card.js

```diff
@@ -13,7 +13,7 @@ const Card = ({ id, img, title, rating }) => {
         key={id} 
         >
         <img src={img} alt={title}></img>
-        <p>{`Rating: ${rating.toFixed(2)}`}</p>
+        <p>{`★ ${rating.toFixed(1)}/10`}</p>
```

Comment:
I think this UX choice is great. I wonder if we had an actual icon we can add alt text to account for accessibility.